src/tlo/methods/enhanced_lifestyle.py

```diff
@@ -825,533 +428,1495 @@ def on_birth(self, mother_id, child_id):
+
+        # create all linear models dictionary for use in both initialisation and update of properties
+        self._models = {
+            'li_urban': {
+                'init': self.rural_urban_linear_model(),
+                'update': self.update_rural_urban_property_linear_model()
+            },
+            'li_wealth': {
+                'init': self.wealth_level_linear_model(),
+                'update': None
+            },
+            'li_low_ex': {
+                'init': self.low_exercise_linear_model(),
+                'update': self.update_exercise_property_linear_model()
+            },
+            'li_tob': {
+                'init': self.tobacco_use_linear_model(),
+                'update': self.update_tobacco_use_property_linear_model()
+            },
+            'li_ex_alc': {
+                'init': self.excessive_alcohol_linear_model(),
+                'update': self.update_excess_alcohol_property_linear_model()
+            },
+            'li_mar_stat': {
+                'init': self.marital_status_linear_model(),
+                'update': self.update_marital_status_linear_model()
+            },
+            'li_in_ed': {
+                'init': EduPropertyInitialiser(self.module, 'li_in_ed'),
+                'update': self.update_education_status_linear_model('li_in_ed')
+            },
+            'li_ed_lev': {
+                'init': EduPropertyInitialiser(self.module, 'li_ed_lev'),
+                'update': self.update_education_status_linear_model('li_ed_lev')
+            },
+            'li_unimproved_sanitation': {
+                'init': self.unimproved_sanitation_linear_model(),
+                'update': self.update_unimproved_sanitation_status_linear_model()
+            },
+            'li_no_clean_drinking_water': {
+                'init': self.no_clean_drinking_water_linear_model(),
+                'update': self.update_no_clean_drinking_water_linear_model()
+            },
+            'li_wood_burn_stove': {
+                'init': self.wood_burn_stove_linear_model(),
+                'update': self.update_wood_burn_stove_linear_model()
+            },
+            'li_no_access_handwashing': {
+                'init': self.no_access_hand_washing(),
+                'update': self.update_no_access_hand_washing_status_linear_model()
+            },
+            'li_high_salt': {
+                'init': self.salt_intake_linear_model(),
+                'update': self.update_high_salt_property_linear_model()
+            },
+            'li_high_sugar': {
+                'init': self.sugar_intake_linear_model(),
+                'update': self.update_high_sugar_property_linear_model()
+            },
+            'li_bmi': {
+                'init': BmiPropertyInitialiser(self.module),
+                'update': self.update_bmi_categories_linear_model()
+            },
+            'li_is_circ': {
+                'init': self.male_circumsision_property_linear_model(),
```

Comment:
```suggestion
                'init': self.male_circumcision_property_linear_model(),
```
Also suggested typo correction definition below